app/jobs/send_complaint_job.rb

```diff
@@ -1,5 +1,28 @@
 class SendComplaintJob < ApplicationJob
   queue_as :send_complaints
 
-  def perform; end
+  def perform(form_builder_payload:)
+    Usecase::Optics::CreateCase.new(
+      optics_gateway: Gateway::Optics.new,
+      presenter: Presenter::Complaint.new(form_builder_payload: form_builder_payload),
+      get_bearer_token: bearer_token
+    ).execute
+  end
+
+  private
+
+  def bearer_token
+    Usecase::Optics::GetBearerToken.new(
+      optics_gateway: Gateway::Optics.new,
+      generate_jwt_token: generate_token
+    )
+  end
+
+  def generate_token
+    Usecase::Optics::GenerateJwtToken.new(
+      url: 'https://uat.icasework.com/token?db=hmcts',
```

Comment:
This should be an ENV var so we can change it to the production endpoint. Doesn't have to be this PR, but if not, we should do it now so we don't forget.

Reply:
agreed